streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -25,20 +33,14 @@
 import org.apache.kafka.streams.processor.TaskId;
 import org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
 /**
  * A StandbyTask
  */
 public class StandbyTask extends AbstractTask {
```

Comment:
Do we need to update/extend `StandbyTasktTest` ?

Reply:
+1. I think I had some tests for this that I neglected to move over. I'll get those in the next patch.